### ISSUE TITLE
Link text and hover color contrast in footer is insufficient

### ISSUE THREAD

**Issue by chalin** [google/docsy#1483]
Note that this problem existed **before we migrated to BSv5**.

> <img width="336" alt="image" src="https://user-images.githubusercontent.com/4140793/224571309-7aa577be-1dfa-4638-b79e-c498cdba047c.png">

Related:

- #1482

**Comment by kumarankit999** [comment]
I can solve this, but I am unable to find that CSS file. Can you help me into this
